Dockerfile

```diff
@@ -18,6 +18,11 @@ RUN test -n "${COMMIT_ID}"
 RUN echo "${COMMIT_ID}" > version-info.txt
 ENV COMMIT_ID "${COMMIT_ID}"
 
+# Set git commit ID
```

Comment:
Comment hasn't been updated from copypasta.
May be worth giving a line about what this variable does

Reply:
lol copy/paste

Done